pkg/api/api.go

```diff
@@ -500,17 +506,19 @@ var FuncMap = template.FuncMap{
 }
 
 // NewScalewayAPI creates a ready-to-use ScalewayAPI client
-func NewScalewayAPI(endpoint, organization, token string) (*ScalewayAPI, error) {
+func NewScalewayAPI(apiEndPoint, accountEndPoint, organization, token string) (*ScalewayAPI, error) {
```

Comment:
Can you run a `make test`, you will need to update the test in ./pkg/commands/command_test.go